ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/mac_security.schema.yml

```diff
@@ -12,17 +12,13 @@ keys:
     keys:
       license:
         type: dict
-        required: true
         keys:
           license_name:
             type: str
```

Comment:
I guess these two keys inside the `license` dict should still be required. Since it does not make sense to have the dangling `license` dict without both of these. The user can still leave out the entire `license` since you remove that requirement in line 15.